src/themes/components/link.ts

```diff
@@ -3,7 +3,7 @@ import { ComponentMultiStyleConfig, SystemStyleObject } from '@chakra-ui/react';
 const linkBaseStyle: SystemStyleObject = {
   textStyle: 'body',
   color: 'blue.700',
-  display: 'flex',
+  display: 'inline-flex',
   flexDirection: 'row',
```

Comment:
I'm wondering if we still need the row direction here.

Reply:
dang, I've only seen that comment after merging. `row` is the default as far as I remember. But defaults can change, doesn't hurt to have it.

Reply:
oki, doki.
Thanks for jumping on the fix ❤️